app/helpers/api_importer.rb

```diff
@@ -0,0 +1,16 @@
+require 'rubygems'
+require 'HTTParty'
+
+class APIImporter
+  include HTTParty
```

Comment:
These methods should live in the job. Added them as a helper means they get included in the views which we don't need. It is a "feature" of rails. 

You could put it in models, but these are just helper methods and we don't need them outside of the job.

Reply:
I'm probably cool with this being it's own class, but not in `helpers`... how about an `app/importers` class?

Reply:
+1

Reply:
I was torn on that.  I'd rather have both methods in a common class (why implement the same class twice?), so I can't put it in the same file as the jobs, because then it would unavailable to the other job.  Should I just move this file out of `app/helpers` and into `app/jobs`?

Reply:
I can move this to an `app/importers` folder.